src/main/java/project/bank/view/JDialog_AddCompAcc.java

```diff
@@ -1,26 +1,22 @@
-package project.bank;
+package project.bank.view;
 
-import java.awt.*;
-import javax.swing.*;
 
+import project.framework.model.*;
+import project.framework.view.AddAccountDialog;
 
+import java.util.Date;
+import java.util.List;
 
-public class JDialog_AddCompAcc extends javax.swing.JDialog
-{
+public class JDialog_AddCompAcc extends AddAccountDialog {
```

Comment:
I didn't look into it too much but when the dialog for the AddCompanyAccount form shows up, all the fields are duplicated. The rest looks on point. Nice Job!